src/components/__tests__/articles/SearchResults.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { BrowserRouter } from 'react-router-dom';
+import { SearchResults } from '../../articles/SearchResults';
+import store from '../../../store';
+
+describe('<SearchResults /> component', () => {
+  const props = {
+    searchResults: [{
+      likes: 1,
+      description: 'Ever wonder how?',
+      rating: 0,
+      comments: ['comment'],
+      title: 'new',
+      slug: 'new_wangonya',
+      reading_time: 'less than 1 minute',
+      reads: 0,
+      views: 310,
+      image: 'wangonya.png',
+      author: 6,
+    }],
+  };
+  const wrapper = mount(
+    <BrowserRouter>
+      <SearchResults
```

Comment:
Why do you include `BrouwserRouter` to mount you wrapper? Also don't you thing using mount may result in using more memory and slowing down of the time to run tests?

Reply:
I had to wrap the component in `BrowserRouter` because I kept getting the error `You should not use <Link> outside a <Router>` while testing. It was an issue with ReactRouter. See [this](https://stackoverflow.com/questions/48640280/you-should-not-use-link-outside-a-router) discussion for more info, and [this](https://stackoverflow.com/a/54519524/9312256) answer for the fix.
I used `mount` because I had to use multiple `.dive()`s in order to get what I wanted. Once you use `.dive()` more than twice, you might as well use `mount` since the same thing is done. The component is very small here so memory isn't an issue. I can confirm that the time needed to run the test is also not significantly high because of it.

Reply:
It looks like a workaround...

Reply:
@FlevianK should I remove it?